pkg/gather/clusterconfig/sap_datahubs_test.go

```diff
@@ -0,0 +1,81 @@
+package clusterconfig
+
+import (
+	"context"
+	"testing"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/runtime/serializer/yaml"
+	dynamicfake "k8s.io/client-go/dynamic/fake"
+)
+
+func TestSAPDatahubs(t *testing.T) {
+	// Initialize the fake dynamic client.
+	datahubYAML1 := `apiVersion: installers.datahub.sap.com/v1alpha1
+kind: DataHub
+metadata:
+    name: example-datahub
+    namespace: example-namespace1
+`
+	datahubYAML2 := `apiVersion: installers.datahub.sap.com/v1alpha1
+kind: DataHub
+metadata:
+    name: example-datahub
+    namespace: example-namespace2
+`
+
+	datahubsResource := schema.GroupVersionResource{Group: "installers.datahub.sap.com", Version: "v1alpha1", Resource: "datahubs"}
```

Comment:
Same here pls.

Reply:
This is exactly the same code as in `master`. The idea here was that if someone introduces a type into the constant, then the tests would happily pass if they also use the constant. However, if each of them has their own hardcoded GVR variable, then they will start failing.